proto/warden/async/v1beta1/future.proto

```
@@ -0,0 +1,55 @@
+syntax = "proto3";
+
+package warden.async.v1beta1;
```

buf (1.47.2)

Files with package "warden.async.v1beta1" must be within a directory "warden/async/v1beta1" relative to root but were in directory "proto/warden/async/v1beta1".

(PACKAGE_DIRECTORY_MATCH)